packages/wrangler/src/metrics/helpers.ts

```diff
@@ -0,0 +1,57 @@
+import os from "node:os";
+import whichPmRuns from "which-pm-runs";
+import { version as wranglerVersion } from "../../package.json";
+
+export function getWranglerVersion() {
+	return wranglerVersion;
+}
+
+export function getPackageManager() {
+	return whichPmRuns()?.name;
+}
+
+// used by "new" metrics
+export function getPlatform() {
+	const platform = os.platform();
+
+	switch (platform) {
+		case "win32":
+			return "Windows";
+		case "darwin":
+			return "Mac OS";
+		case "linux":
+			return "Linux";
+		default:
+			return `Others: ${platform}`;
+	}
+}
+
+// used by "old" metrics
+export function getOS() {
+	return process.platform + ":" + process.arch;
+}
+
+export function getOSVersion() {
+	return os.version();
+}
+
+export function getNodeVersion() {
+	return process.version;
+}
+
+export function getConfigFileType(configPath: string | undefined) {
```

Comment:
There's an existing helper for this—`configFormat` in `config/index.ts`

Reply:
haha there wasn't when i added that! will use `configFormat` 👍

Reply:
This helper seems like a 1:1 passthrough?

Reply:
5fcef96670a2fa2e897f868dbce666a7efbe7b61